### DISCUSSION TITLE
SHA256SUMS.asc for version 5.0.1

### DISCUSSION THREAD

**Discussion by tnc-12** [OneKeyHQ/app-monorepo#5370] [Q&A] [answered]
Hello,
version 5.0.1 is available by default as Linux download:
https://web.onekey-asset.com/app-monorepo/v5.0.1/OneKey-Wallet-5.0.1-linux-x86_64.AppImage
but the verification file:
https://web.onekey-asset.com/app-monorepo/assets/SHA256SUMS.asc
still shows version 4.24.0.
Where is the verification file for version 5.0.1?
is it the latest version ? because here it still says the latest version is 4.24.0.
I'm confused...

**Comment by steinvu** [comment]
Yeah, this is missing for 5.0.1.

Please add asap. Thanks

**Reply by originalix** [reply, marked as answer]
Thank you for your feedback. This file was missing in version 5.0.1. Please upgrade to the latest version 5.1.2.

**Reply by steinvu** [reply]
OK and where is the checksum file for this 5.1.2 version? Still can't find it atm.

**Reply by revan-zhang** [reply]
Hi @steinvu Sorry for the late response.For every version you can get the checksum file from link `https://web.onekey-asset.com/app-monorepo/{version}/SHA256SUMS.asc`

**Comment by revan-zhang** [comment]
Thanks, checksum files stored at `https://web.onekey-asset.com/app-monorepo/{version}/SHA256SUMS.asc` for every version.
